### PR TITLE
Issue479 modular data bus

### PR DESCRIPTION
**Pull request by MirkoEngelpracht**
Good evening everybody,

I finished the modular data bus connectors required to solve #407. I've compared them with the data bus connectors suggested in #354 and #355, respectively. However, these interfaces are not suitable for #407 because we develop a heat pump model that is vectorised (i.e. a high dimensional one). Therefore, we need bus connectors that include various variables presented as vectors.

### CODE REVIEW

**Comment by tstorek** [comment]
Thank your for your contribution. I will review as soon as possible,

**Comment by MirkoEngelpracht** [comment]
@tstorek, thank you. I'm looking forward for your review!

File: AixLib/Controls/Interfaces/ModularExpansionValveControlBus.mo [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+  // Definition of parameters describing controlling system in general
+  //
+  parameter Boolean extConVal = false
+    "= true, if external signal is used for expansion valves"
+    annotation(Dialog(tab="Expansion Valves",group="General"));
+
+  // Definition of variables describing expansion valves
+  //
+  parameter WorkingVersion.Controls.Choices.ControlVariable controlVariableVal=
```
Comment (tstorek) [inline AixLib/Controls/Interfaces/ModularExpansionValveControlBus.mo]: Path does not seem to be correct for final version

**Comment by MirkoEngelpracht** [comment]
@tstorek, thank you very much for your review :) I've changed all errors!

**Comment by tstorek** [comment]
This closes issue #479

**Pull request merged**